packages/main/src/RPA/Excel/Files.py

```diff
@@ -1206,6 +1219,33 @@ def append_worksheet(self, name=None, content=None, header=False, start=None):
         if header and is_empty:
             sheet.append(columns)
 
+        if formatting_as_empty:
+            self._append_on_first_empty_based_on_values(content, columns, sheet)
+        else:
+            self._default_append_rows(content, columns, sheet)
+
+        self.active = sheet_name
+
+    def _append_on_first_empty_based_on_values(self, content, columns, sheet):
+        first_empty_row = None
```

Comment:
what the... PyCharm is unhappy because of those kind of things ? ....

Reply:
Was, because of the `int` typing we introduced below (to ensure the dev is aware that no matter what this var should have an `int` value in the end):
![Uploading Screenshot 2022-08-11 at 14.00.54.png…]()